client/src/featureform/get.py

```diff
@@ -74,27 +75,40 @@ def get_resource_info(stub, resource_type, name):
     except grpc._channel._MultiThreadedRendezvous:
         print(f"{resource_type} not found.")
 
-def get_feature_variant_info(stub, name, variant):
+def get_feature_variant_info(stub, name, variant, display=True):
```

Comment:
is this display=True a common idiom in Python?

Reply:
Do you think verbose would be a more sensible parameter name? This toggles whether the get function automatically prints resource info

Reply:
Why would people want display to be true? If you're using it in the CLI then the CLI logic should display it, if not then you should just print the object. So we can have whatever is returned be easy to print